portal-ui/src/screens/Console/Account/ChangeUserPasswordModal.tsx

```diff
@@ -67,9 +67,14 @@ const ChangeUserPassword = ({
       return;
     }
     setLoading(true);
+ 
+    if (newPassword.length < 8) {
+      setModalErrorSnackMessage("Passwords must be at least 8 characters long");
```

Comment:
Please set loading to false after showing the error:
```suggestion
      setModalErrorSnackMessage("Passwords must be at least 8 characters long");
      setLoading(false);
```